### PR TITLE
Task: Notification (continue)

### PR DESCRIPTION
**Pull request by HaiHaChan**

### CODE REVIEW

File: app/Repositories/Eloquents/NotificationEloquentRepository.php [hunk 1]
```diff
@@ -84,11 +89,27 @@ public function find($id)
...
+                return $record->update($data);
+            } else {
+                return null;
+            }
+            
         } catch (Exception $e) {
             return null;
         }
     }
+
```
Comment (framgiaciv3) [inline app/Repositories/Eloquents/NotificationEloquentRepository.php:102]: **Reporter**: CHECKSTYLE</br>**Rule**: Squiz.WhiteSpace.ControlStructureSpacing.SpacingBeforeClose</br>**Severity**: error</br>**File**: app/Repositories/Eloquents/NotificationEloquentRepository.php</br>Blank line found at end of control structure

File: app/Http/Controllers/User/VoteController.php [hunk 2]
```diff
@@ -14,12 +15,17 @@ class VoteController extends Controller
 
     protected $review;
 
+    protected $notifiaction;
+
     public function __construct(
         ReviewBookRepository $review,
-        VoteRepository $vote
+        VoteRepository $vote,
+        NotificationRepository $notifiaction
```
Comment (BunnyPi04) [inline app/Http/Controllers/User/VoteController.php]: viết sai chính tả
notification
dưới nữa

File: config/model.php [hunk 3]
```diff
@@ -13,6 +13,12 @@
     	'user' => 'App\Eloquent\User',
         'review' => 'App\Eloquent\Review',
     	'vote' => 'App\Eloquent\Vote',
```
Comment (BunnyPi04) [inline config/model.php]: tab lại nhá

File: resources/assets/user/js/main.js [hunk 4]
```diff
@@ -540,6 +540,7 @@
             data: f.serialize(),
         })
         .done(function(data) {
+            console.log(data);
```
Comment (BunnyPi04) [inline resources/assets/user/js/main.js]: xóa console.log()

**Pull request merged**